weaver/tests/network-setups/fabric/dev/scripts/setupCC.sh

```diff
@@ -4,7 +4,7 @@
 
 directory=$(dirname $0)
 
-CACTI_VERSION=v2.0.0
+CACTI_VERSION="v$(head -n 1 $PWD/../../../../core/network/fabric-interop-cc/contracts/interop/VERSION)"
```

Comment:
@dnakashima This seems to hardcode the behavior in  way that forces people to run the script only from a specific given sub-directory.

What that means is that if I ran this script from the project root like 
```sh
$ ./weaver/tests/network-setups/fabric/dev/scripts/setupCC.sh
```

Then it would crash with some error that it can't find the `VERSION` because the $PWD would evaluate to the project  root and then your `../../../../` would just point to some random, non-existent directory path on my file-system outside of the cacti project sources.

What I recommend to fix this is to anchor the paths to the path of the script itself the way the `ci.sh`  script does it:

```sh
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

After which you can reach the version file like `"$SCRIPT_DIR/../../../../core/network/fabric-interop-cc/contracts/interop/VERSION'` (or something similar) and with this way the script will function correctly regardless of which directory my shell was in at the time of the script invocation.

I wrote all this code just from memory so there might be issues with it, but please let me know what you think!

Reply:
@petermetz Thank you for your comment. I did not consider the directory where setupCC.sh is called. So I modify how to get the path of VERSION file.

Reply:
@dnakashima My pleasure! Thank you for the improvements!